product_configurator/static/src/js/form_widgets.esm.js

```diff
@@ -0,0 +1,33 @@
+/* @odoo-module */
+import {FormController} from "@web/views/form/form_controller";
+import {onMounted} from "@odoo/owl";
+import {patch} from "@web/core/utils/patch";
+
+patch(FormController.prototype, {
+    setup() {
+        super.setup(...arguments);
+        onMounted(() => {
+            var form_element = this.rootRef.el;
+            var self = this;
+            if (
+                self.model.config.resModel === "product.product" &&
+                self.model.config.context.custom_create_variant
+            ) {
+                var buttons = form_element.querySelector(
+                    ".o_control_panel_main_buttons"
+                );
+                var createButtons = buttons.querySelectorAll(".o_form_button_create");
+                createButtons.forEach((button) => {
+                    button.style.display = "none";
+                });
+            }
+        });
+    },
+    async beforeExecuteActionButton(clickParams) {
+        if (clickParams.special === "no_save") {
+            delete clickParams.special;
+            return true;
+        }
+        return super.beforeExecuteActionButton(...arguments);
+    },
```

Comment:
It is the code for performing something different than special=cancel. So, it will not save the form and also keep it open. If we use cancel, the form will also be closed. 
This has been used then we do the Reconfigure.

Reply:
I see. Thanks for taking the time to reply